mmdet3d/datasets/kitti_dataset.py

```diff
@@ -32,12 +32,18 @@ class KittiDataset(Det3DDataset):
             - 'LiDAR': Box in LiDAR coordinates.
             - 'Depth': Box in depth coordinates, usually for indoor dataset.
             - 'Camera': Box in camera coordinates.
-        filter_empty_gt (bool): Whether to filter the data with empty GT.
```

Comment:
Remind that some configs in specific methods may change `filter_empty_gt` and some other filter arguments. For example,  `filter_empty_gt=False` in configs/mvxnet/mvxnet_fpn_dv_second_secfpn_8xb2-80e_kitti-3d-3class.py. Please check them.